README.md

```diff
@@ -110,7 +110,9 @@ Validator                               | Description
 **isHash(str, algorithm)**              | check if the string is a hash of type algorithm.<br/><br/>Algorithm is one of `['md4', 'md5', 'sha1', 'sha256', 'sha384', 'sha512', 'ripemd128', 'ripemd160', 'tiger128', 'tiger160', 'tiger192', 'crc32', 'crc32b']`
 **isHexadecimal(str)**                  | check if the string is a hexadecimal number.
 **isHexColor(str)**                     | check if the string is a hexadecimal color.
+**isHSL(str)**                          | check if the string is an HSL (hue, saturation, lightness) color.
```

Comment:
@kurisu-gh, I think should not be there has there is the one below. Or, take the one below to be in this place.

Reply:
Thank you for catching this! Will take care of it.

Reply:
Fixed now, thanks again.